waiter/integration/waiter/kubernetes_scheduler_integration_test.clj

```diff
@@ -337,3 +337,50 @@
                     :interval 15
                     :timeout timeout-secs)))))
         (log/warn "skipping test as INTEGRATION_TEST_BAD_IMAGE is not specified")))))
+
+(deftest ^:parallel ^:integration-fast test-kubernetes-reverse-proxy-sidecar-headers
+  (testing-using-waiter-url
+    (when (using-k8s? waiter-url)
+      (when-let [proxy-options (get-k8s-proxy-options waiter-url)]
+        (let [{:keys [service-id] :as response}
+              (make-request-with-debug-info
+                {:x-waiter-distribution-scheme "simple"
```

Comment:
Why do we need `simple` distribution for this test?

Reply:
You are right. There is no need for that here